po3/bg.po

```diff
@@ -58,31 +58,31 @@ msgstr "1 000 км/с"
 msgid "10c"
 msgstr "10 c"
 
-# Windows, 10x time speed
+# Mac, Make time go faster
```

Comment:
please do not edit the comment either, this applies to the rest of the PR

Reply:
I did a sync with celestia_ui.pot. These changes were made by Poedit.

Reply:
can you enable editing from maintainers for this PR, I'll use my too to edit it back

Reply:
Done.